packages/css/src/components/search-bar/search-bar.scss

```diff
@@ -1,10 +1,44 @@
+@import "../../../../sources/src/styling/variables";
+
 $dso-search-bar-height: $u5;
 $dso-search-bar-invalid-input-border-color: $dso-danger-color;
+$dso-search-bar-invalid-border-color: $dso-danger-color;
+$dso-search-bar-focus-border-width: 1px;
 
 .dso-search-bar {
   display: flex;
   margin-bottom: 16px;
 
+  .form-group & {
```

Comment:
Alle styling die met het formulier te maken heeft doen we in een losse issue.

Reply:
Hoewel ik na het reviewen wel begrijp waar het vandaan komt.. Ik kom even bij je overleggen.

Reply:
fixed